workspaces/ui-v2/src/optic-components/pages/docs/EditContributionsButton.tsx

```diff
@@ -53,7 +53,10 @@ export function EditContributionsButton() {
       <CommitMessageModal
         open={commitModalOpen}
         onClose={() => setCommitModalOpen(false)}
-        onSave={save}
+        onSave={(commitMessage: string) => {
```

Comment:
This is for the document version of the commit message modal - interestingly we need to not automatically close the diff modal, otherwise what happens is that it remounts the PromptNavigateAway modal and since the history.navigate is outside of the react render cycle this prompt still shows up